### PR TITLE
Ch6898/rest format bridge status

### PR DESCRIPTION
**Pull request by karthikbr82**

### CODE REVIEW

File: callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsHttp2Client.java [hunk 1]
```diff
@@ -4,135 +4,152 @@
...
-	    Request request = new Request.Builder()
-        .url(EVENTS_BASE_URL+url)
-        .addHeader("Accept", "application/json")
-	    .addHeader("Accept-Charset", "utf-8")
-        .post(msg)
-        .build();
-	    logger.debug("sending request "+body);
-	    send(request, listener);
+		RequestBody msg = RequestBody.create(JSON, body);
+		Request request = new Request.Builder().url(url).addHeader("Accept", "application/json").addHeader("Accept-Charset", "utf-8")
```
Comment (eljasala) [inline callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsHttp2Client.java]: Do we need Accept headers?

File: callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsBridgeKeepAliveManager.java [hunk 2]
```diff
@@ -139,45 +147,49 @@ public void run() {
...
+	 *            the bridge id
+	 * @param token
+	 *            the token
+	 * @param httpClient
+	 *            the http client
+	 * 
 	 */
 	private void sendKeepAliveBridgeMessage(int appId, String bridgeId, String token, final CallStatsHttp2Client httpClient) {
 		long apiTS = System.currentTimeMillis();
-		BridgeKeepAliveMessage message = new BridgeKeepAliveMessage(appId, bridgeId, CallStatsConst.CS_VERSION, apiTS, token);
```
Comment (eljasala) [inline callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsBridgeKeepAliveManager.java]: Do we send the version somewhere else?

Comment (karthikbr82) [reply callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsBridgeKeepAliveManager.java]: may be, need consultation with @stenhagglund

File: callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsHttp2Client.java (hunk 1 above)
Comment (karthikbr82) [reply callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsHttp2Client.java]: yes.

**Pull request closed without merging**